pkg/edgeview/src/edgeview_test.go

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2024 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	"path"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
```

Comment:
https://github.com/lf-edge/eve/pull/3435#discussion_r1311796168 ;-)

Reply:
it's because I know how to use `testify` and I don't know how to use `gomega` :smile: 
but if you want I'll learn how to use `gomega` and switch to it. is it better than `testify`?

Reply:
Brining go-chi was useful, because they have route parameter parser built-in, so we didn't have to build our own. The only reason why we left parts of Gorilla then was because we used ws communication (happy to restart the conversation of refactoring it). Rest was completely replaced by go-chi. Basically I'm asking the same question here: what's the benefit of testify over gomega and why do we want to have two testing frameworks in the repo

Reply:
> it's because I know how to use testify and I don't know how to use gomega 😄
but if you want I'll learn how to use gomega and switch to it. is it better than testify?

I'm not insisting on rewriting it, if you think it's better to use testify, let's use testify I have no problem with that

Reply:
https://gorilla.github.io/blog/2023-07-17-project-status-update/

Reply:
@uncleDecart I've rewritten the tests with gomega - it's not much of a difference, so it makes sense to me to stick with one testing framework

Reply:
btw
```
grep -r "github.com/stretchr/testify/assert" pkg/ | grep "_test.go" | wc -l
26
```

Reply:
Thank you very much for change! Missed the fact that you bring testify to edge_view, not pillar, didn't know we were using testify, I think it's a good raise on LF-Edge call